proxy/tcpproxy/userspace.go

```diff
@@ -33,16 +39,18 @@ func (r *remote) inactivate() {
 	r.inactive = true
 }
 
-func (r *remote) tryReactivate() {
+func (r *remote) tryReactivate() error {
```

Comment:
@xiang90 You are right. Just changed it back and wrap the whole function in goroutine as you suggested. PTAL.